### DISCUSSION TITLE
Need Help Updating FFmpeg on Home Assistant OS (Raspberry Pi 4)

### DISCUSSION THREAD

**Discussion by gladrun** [blakeblackshear/frigate#8369] [Ask A Question]
I'm currently running a Home Assistant OS on my Raspberry Pi 4, and I've been using Frigate for detection. Recently, I stumbled upon this log message: "RTP H.264 NAL unit type 26 is not implemented. Update your FFmpeg version to the newest one from Git. If the problem still occurs, it means that your file has a feature which has not been implemented."

I'm a bit lost on how to update FFmpeg to resolve this issue. Can anyone guide me through the process? Your help would be greatly appreciated!

**Comment by blakeblackshear** [comment]
This message is almost certainly not your problem. I would open an actual support issue instead of a q&a discussion and provide a full set of logs.

**Reply by gladrun** [reply]
I had set wrong resolution of cameras. I stopped getting these log message after I had set them correctly.
